timeseries/blueflood/blueflood_test.go

```diff
@@ -16,607 +16,240 @@ package blueflood
 
 import (
 	"fmt"
-	"net/http"
 	"testing"
 	"time"
 
 	"github.com/square/metrics/api"
 	"github.com/square/metrics/testing_support/assert"
-	"github.com/square/metrics/testing_support/mocks"
-	"github.com/square/metrics/timeseries"
-	"github.com/square/metrics/util"
 )
 
-func Test_Blueflood(t *testing.T) {
-	timerange, err := api.NewTimerange(12000, 13000, 1000)
-	if err != nil {
-		t.Fatalf("invalid testcase timerange")
-		return
-	}
-
-	graphite := mocks.FakeGraphiteConverter{
-		MetricMap: map[util.GraphiteMetric]api.TaggedMetric{
-			util.GraphiteMetric("some.key.graphite"): {
-				MetricKey: api.MetricKey("some.key"),
-				TagSet:    api.TagSet{"tag": "value"},
-			},
-		},
-	}
+const day = 24 * time.Hour
 
-	defaultClientConfig := Config{
-		BaseURL:                 "https://blueflood.url",
-		TenantID:                "square",
-		Ttls:                    make(map[string]int64),
-		Timeout:                 time.Millisecond,
-		FullResolutionOverlap:   0,
-		GraphiteMetricConverter: &graphite,
-	}
-	// Not really MIN1440, but that's what default TTLs will get with the Timerange we use
-	defaultQueryURL := "https://blueflood.url/v2.0/square/views/some.key.graphite?from=12000&resolution=MIN1440&select=numPoints%2Caverage&to=14000"
+var resolutionFull = Resolution{
+	Name:           "FULL",
+	Resolution:     30 * time.Second,
+	FirstAvailable: 0,
+	// TimeToLive is open, so setting it to 1 day exactly means queries from -1d to now will need 5min.
```

Comment:
Various adjustments have made this comment inaccurate. I've change it back to `1 * day`.